tests/testthat/test_PlainTextParam.R

```diff
@@ -12,42 +12,80 @@ test_that("storeResults,loadResults,PlainTextParam,MsBackendMzR works", {
     ## Loading data again
     b2 <- loadResults(object = MsBackendMzR(), param)
     expect_true(inherits(b2, "MsBackendMzR"))
-    b <- dropNaSpectraVariables(b) #the function does this to be robust, is it a problem ? i should mention it in the doc
```

Comment:
yes, that's a good point. We should mention this. but this will be specific for the `storeResults,MsBackendMzR` - for other backends we might not need that.